vscode tizen extension user docs update for Rpk app (#1703)
Samsung/tizen-docs: docs/application/vscode-ext/Tizen/rpk.md

> # Rpk (Resource Package) Application Development

Reviewer: ```suggestion
# RPK (Resource Package) Application Development
```

The VS Code extension installation can be done in two different ways. Refer to [VS Code extension installation](dotnet.md) for details. 

## Develop application

> The following sections explain how to use Visual Studio Code Extension for Tizen to develop your applications.

Reviewer: ```suggestion
The following sections explain how to use Visual Studio Code Extension for Tizen to develop RPK applications.
```

> ### Create Tizen Rpk project

Reviewer: ```suggestion
### Create Tizen RPK project
```

> To create a Tizen RPK project:

Reviewer: ```suggestion
To create a Tizen RPK project, follow these steps:
```

> To create a Tizen Rpk project:

Reviewer: ```suggestion
To create a Tizen RPK project:
```

9. Enter the project name.

   ![project name](media/rpk_project_name.png)

### Build your project

1. Open the **Command Palette** and enter Tizen build. Select **Tizen: Build Tizen project**.

   ![Build project](media/rpk_build.png)

2. Review the build results in the output window, and check the location of the package file (.rpk).

   ![Review result](media/rpk_build_result.png)

### Deploy and run your application in emulator

1. To launch the Tizen Emulator Manager, open the **Command Palette** and enter Tizen Emulator. Select **Tizen: Launch Tizen Emulator Manager**.

   ![Launch emulator](media/rpk_deploy.png)

2. Create and launch an emulator instance in the Emulator Manager.

3. To deploy your application to the target, enter Tizen Install in the **Command Palette** and select **Tizen: Install Tizen application**.

   ![Install application](media/rpk_install_app.png)

   ![Install application](media/rpk_install_app_result.png)

4. To run the application on the emulator, enter run in the **Command Palette** and select **Tizen: Run Tizen application**.

   ![Run application](media/rpk_run_app1.png)

   ![application](media/rpk_run_app2.png)

   > [!NOTE] 

>    > "Resource project is Installed, But Resource project cannot be launched by Run" An info popup will be displayed.

Reviewer: ```suggestion
   > An info popup will be displayed, showing the message "Resource project is Installed, but resource project cannot be launched by Run".
```

> # Tizen workspace yaml Validation

Reviewer: I think second heading is more appropriate here, as an md file can only have one First heading or Title on Tizen Docs.
```suggestion
## Tizen workspace YAML validation
```

> 1. Create Tizen project using command **Tizen: Create Tizen project**..

Reviewer: ```suggestion
1. Create Tizen project using command **Tizen: Create Tizen project**.
```

> 2. Open tizen_workspace.yaml file, append character d to type:hybrid. After that a red line displays under the word 'hybridd' when the mouse is hovered on it. An error message will also be displayed.

Reviewer: ```suggestion
2. Open the tizen_workspace.yaml file and append the character 'd' to type:hybrid. After that, a red line displays under the word 'hybridd' when the mouse is hovered on it. An error message will also be displayed.
```

  
   ![worksplace validation](media/rpk_tizen_workspace_yaml_open.png)

   ![worksplace validation](media/rpk_tizen_workspace_yaml_validation.png)

> # Dotnet Workload Installation

Reviewer: Same suggestion as line 82.
```suggestion
## Dotnet workload installation
```

> 1. Dotnet workload installation will be installed on vscode activation OR by using command **Tizen: Install Dotnet Workload**..

Reviewer: ```suggestion
1. Dotnet workload installation will be installed on VS Code activation OR by using the command **Tizen: Install Dotnet Workload**..
```

> 1. Dotnet workload installation will be installed on VS Code activation OR by using the command **Tizen: Install Dotnet Workload**..

Reviewer: ```suggestion
1. Dotnet workload installation will be installed on VS Code activation OR by using the command **Tizen: Install Dotnet Workload**.
```

> 2. For Windows OS, Run vscode as Administrator.

Reviewer: ```suggestion
2. For Windows OS, Run VS Code as Administrator.
```

> 3. For Ubuntu/Mac OS, Enter system password as it installation required sudo permission OR Press Esc to skip installation.

Reviewer: ```suggestion
3. For Ubuntu/Mac OS, enter system password as the installation requires sudo permission. Or you can press Esc to skip installation.
```

> 3. For Ubuntu/Mac OS, enter system password as the installation requires sudo permission. Or you can press Esc to skip installation.

Reviewer: ```suggestion
3. For Ubuntu/macOS, enter system password as the installation requires sudo permission. Or you can press Esc to skip installation.
```

  
   ![dotnet workload installation](media/dotnet_workload_install_password.png)

   ![dotnet workload installation](media/dotnet_workload_install_password_enter.png)
   
   ![dotnet workload installation](media/dotnet_workload_install_success.png)
   
   > [!NOTE] 

>    > dotnet workload installion requires .NET Core SDK (.NET6 or higher).

Reviewer: ```suggestion
   > Dotnet workload installation requires .NET Core SDK (.NET6 or higher).
```

>    > Download and Install Latest .NET Core SDK from <https://www.microsoft.com/net/download/>

Reviewer: ```suggestion
   > Download and install the latest version of .NET Core SDK from <https://www.microsoft.com/net/download/>
```
